src/pages/home.jsx

```diff
@@ -1,6 +1,15 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import { UserContext } from '../context/userContext';
+import { useHistory } from 'react-router-dom';
 
 export const Home = () => {
+  const user = useContext(UserContext);
+
+  const history = useHistory();
+  if (!user) {
+    history.push('/login');
+  }
```

Comment:
this page when reload,
the user value comes as null, and it directs you to the login page,
so this issue needs to handle,
try to check the user context, when you fix, the bug may be in this context.